lsr_role2collection/collection_readme.txt

```diff
@@ -0,0 +1,47 @@
+﻿# Linux System Roles Ansible Collection

```

Comment:
shouldn't it be called ...md ? it is in Markdown format, not some random text, right?
(If you give it the right suffix, tools like editors will be able to treat it correctly, e.g. use proper syntax highlighting.)

Reply:
I think the actual README.md is generated from this

Reply:
@richm, sure, but this file itself is not in the markdown format? If it is, it is preferable to give it the .md suffix.
(Or it could be called ...`.md.in`)

Reply:
how about `collection_readme.md`?   I don't think `.md.in` is appropriate - it isn't a jinja2 template or something like that.